src/components/Episodes/reducer.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import {NEXT_EPISODE, PREV_EPISODE, SET_OVERLAY_ACTIVE, SET_ACTIVE_INDEX} from "./";
+
+export default function reducer(state, action){
+	const {
+		type, 
+		value
+	} = action;
+
+	switch(type){
+		case NEXT_EPISODE:
+			return {
+				...state,
+				activeIndex: state.activeIndex+1
+			}
+		case PREV_EPISODE:
+			return {
+				...state,
+				activeIndex: state.activeIndex-1
+			}
+		case SET_ACTIVE_INDEX:
+			return {
+				...state,
+				activeIndex: value
+			}
+		case SET_OVERLAY_ACTIVE:
+			return {
+				...state,
+				overlayActive: value
+			}
+		default:
+			throw new Error("Invalid action requested.");
+			break;
```

Comment:
Nice - good shout, although three points!
- This will cause everything to fail and crash - if that's what you want then keep as is, if you just want to alert yourself, then `console.error()` is also available
- You don't need an instance of `Error` to throw something - you can just do a string unless you specifically want to use stuff from the `Error` class.
- Whenever you're throwing developer errors like this - make debugging as helpful as possible by giving yourself an indication of what went wrong - ie 

```
throw `Invalid action: ${type} requested with a value of ${value}`;
```